src/icons/AnonFace.tsx

```diff
@@ -0,0 +1,76 @@
+import classnames, {
+  boxShadow,
+  boxShadowColor,
+  stroke,
+  strokeWidth,
+} from 'classnames/tailwind'
+
+const pathStyle = classnames(
+  strokeWidth('stroke-2'),
+  boxShadow('shadow-lg'),
```

Comment:
Do every path needs shadow? On the demo video I can't see it

Reply:
It's really slight and yep, moved it to the svg